test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -122,6 +122,7 @@ public void GenerateSchema_DedupsEnumValues_IfEnumTypeHasDuplicateValues()
 
         [Theory]
         [InlineData(typeof(IDictionary<string, int>), "integer")]
+        [InlineData(typeof(IDictionary<EmptyIntEnum, int>), "integer")]
```

Comment:
Does this test fail without your other code change?

Reply:
Yes, it fails.